pkg/storaged/resize.jsx

```diff
@@ -122,18 +129,21 @@ export function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
         }
     }
 
-    function lvm_resize() {
-        if (size != lvol.Size)
-            return lvol.Resize(size, { });
-        else
+    function lvol_or_part_resize() {
+        if (size != orig_size) {
+            if (lvol_or_part.iface == "org.freedesktop.UDisks2.LogicalVolume")
+                return lvol_or_part.Resize(size, { });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19385-20230928-071939-cd48ea23-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)